### ISSUE TITLE
Dataset license

### ISSUE THREAD

**Issue by BlazejOstrowski** [morrisfranken/glyphreader#3]
Hi, Is there any license to use your dataset in this project? Best regards

**Comment by morrisfranken** [comment]
Hi @Gango76 , thanks for asking, the dataset was introduced with the paper titled "Automatic Egyptian Hieroglyph Recognition
by Retrieving Images as Texts" (https://dl.acm.org/doi/pdf/10.1145/2502081.2502199). If you plan to use it for research purposes , kindly ensure that you appropriately reference the aforementioned paper.

**Comment by BlazejOstrowski** [comment]
I see. So, it's sufficient if I refer to this article to use the database for research work? Thank you very much

**Comment by morrisfranken** [comment]
Yes that will be sufficient. Looking forward seeing your experiments and results!

**Comment by BlazejOstrowski** [comment]
Thank you very much
